submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionDraftApiTest.kt

```diff
@@ -51,21 +45,29 @@ internal class SubmissionDraftApiTest(tempFolder: TemporaryFolder) : BaseIntegra
 
         @Test
         fun `get draft submission when draft does not exist but submission does`() {
+            val pageTab = jsonObj { "accno" to "ABC-123"; "type" to "Study" }.toString()
```

Comment:
Not sure about this one-liner TBH. I think it looks better in separated lines and it's clearer that's a two lines draft but up to you.